server/service/src/sync/translations/mod.rs

```diff
@@ -208,31 +215,42 @@ impl PushTranslateResult {
     pub(crate) fn upsert(
         changelog: &ChangelogRow,
         table_name: &str,
-        data: serde_json::Value,
+        record_data: serde_json::Value,
     ) -> Self {
-        Self::PushRecord(vec![RemoteSyncRecordV5 {
-            sync_id: changelog.cursor.to_string(),
-            record: CommonSyncRecordV5 {
+        Self::PushRecord(vec![PushSyncRecord {
+            cursor: changelog.cursor,
+            record: CommonSyncRecord {
                 table_name: table_name.to_string(),
                 record_id: changelog.record_id.clone(),
-                action: SyncActionV5::Update,
-                data,
+                action: SyncAction::Update,
+                record_data,
             },
         }])
     }
     pub(crate) fn delete(changelog: &ChangelogRow, table_name: &str) -> Self {
-        Self::PushRecord(vec![RemoteSyncRecordV5 {
-            sync_id: changelog.cursor.to_string(),
-            record: CommonSyncRecordV5 {
+        Self::PushRecord(vec![PushSyncRecord {
+            cursor: changelog.cursor,
+            record: CommonSyncRecord {
                 table_name: table_name.to_string(),
                 record_id: changelog.record_id.clone(),
-                action: SyncActionV5::Delete,
-                data: Default::default(),
+                action: SyncAction::Delete,
+                record_data: Default::default(),
             },
         }])
     }
 }
 
+pub(crate) enum PushTranslationType {
+    // When omSupply remote is pushing to og mSupply central
+    Legacy,
+    // When omSupply remote is pushing to omSupply central
+    #[allow(dead_code)]
+    OmSupplyRemoteSitePush,
+    // When omSupply ceantral is pushing to omSupply remote
+    // this is when omSupply central is responding to pull request from omSupply remote
```

Comment:
Should comment be after the  `OmSupplyCentralSitePush` a bit confused if this all relates to    `OmSupplyRemoteSitePush`?

Reply:
Also central (: